nss-cohort-36/bangazon-client-virtual-magicians-1
# created basic navbar

# Description
Created a NavBar. Does not do user authentication yet!!!
Closes https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/52
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-navbar-setup`
4. Run `npm install` and then `npm start`

# Checklist:
